extensions/amp-ima-video/0.1/test/test-amp-ima-video.js

```diff
@@ -0,0 +1,893 @@
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {createIframePromise} from '../../../../testing/iframe';
+import {adopt} from '../../../../src/runtime';
+import * as imaVideoObj from '../../../../ads/google/imaVideo';
+import * as sinon from 'sinon';
+import '../amp-ima-video';
+
+adopt(window);
+
+describe('amp-ima-video', () => {
+  const srcUrl = 'http://rmcdn.2mdn.net/Demo/vast_inspector/android.mp4';
+  const adTagUrl = 'https://pubads.g.doubleclick.net/gampad/ads?sz=640x480&iu=/124319096/external/ad_rule_samples&ciu_szs=300x250&ad_rule=1&impl=s&gdfp_req=1&env=vp&output=vmap&unviewed_position_start=1&cust_params=deployment%3Ddevsite%26sample_ar%3Dpremidpost&cmsid=496&vid=short_onecue&correlator=';
+
+  let sandbox;
+
+  beforeEach(() => {
+    sandbox = sinon.sandbox.create();
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  it('adds ad container', () => {
+    return createIframePromise().then(iframe => {
+      const div = document.createElement('div');
+      div.setAttribute('id', 'c');
+      iframe.doc.body.appendChild(div);
+
+      imaVideoObj.imaVideo(iframe.win, {
+        width: 640,
+        height: 360,
+        src: srcUrl,
+        tag: adTagUrl,
+      });
+      const video = iframe.doc.body.querySelector('#ima-content-player');
+      expect(video).not.to.be.undefined;
+    });
+  });
+
+  it('adds video element', () => {
+    return createIframePromise().then(iframe => {
+      const div = document.createElement('div');
+      div.setAttribute('id', 'c');
+      iframe.doc.body.appendChild(div);
+
+      imaVideoObj.imaVideo(iframe.win, {
+        width: 640,
+        height: 360,
+        src: srcUrl,
+        tag: adTagUrl,
+      });
+      const video = iframe.doc.body.querySelector('#ima-ad-container');
+      expect(video).not.to.be.undefined;
+    });
+  });
+
+  it('handles click', () => {
+    return createIframePromise().then(iframe => {
+      const div = document.createElement('div');
+      div.setAttribute('id', 'c');
+      iframe.doc.body.appendChild(div);
+
+      imaVideoObj.imaVideo(iframe.win, {
+        width: 640,
+        height: 360,
+        src: srcUrl,
+        tag: adTagUrl,
+      });
+      const bigPlayDivMock = {
+        style: {
+          display: '',
+        },
+        removeEventListener: function() {},
+      };
+      const removeEventListenerSpy = sandbox.spy(
+          bigPlayDivMock, 'removeEventListener');
+      const adDisplayContainerMock = {initialize: function() {}};
+      const initSpy = sandbox.spy(adDisplayContainerMock, 'initialize');
+      const videoPlayerMock = {load: function() {}};
+      const loadSpy = sandbox.spy(videoPlayerMock, 'load');
+      //const playAdsSpy = sandbox.spy(imaVideoObj, 'playAds');
```

Comment:
give this a try: `const playAdsFunc = imaVideoObj.playAds; const playAdsSpy = sandbox.spy(playAdsFunc);` ( I think issue is that `imaVideoObj` is not a real object but rather a grouping of exported methods created by the `import` syntax and I have no idea how `import * as x` transpiles into ES5)